### PR TITLE
Dev Portals Blog Post

### PR DESCRIPTION
**Pull request by meagancojocar**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Review by scottslowe — CHANGES_REQUESTED** [review]
One formatting change needed, otherwise looks good

File: themes/default/content/blog/building-developer-portals/index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+2. Organization Templates: Pulumi Cloud now supports registering your own custom templates for your organization which will be made available in the New Project Wizard. This enables platform teams to standardize infrastructure creation and distribute those best practices where developers are already spending their time.
+3. Pulumi Backstage Plugin: For teams using Backstage, they can now integrate Pulumi directly within their Backstage instance, via support for a new Pulumi plugin as well as new Pulumi scaffolder actions.
+4. `pulumi new` support for private templates: By default, `pulumi new` provides a number of templates provided by Pulumi, but now your organization can also use private templates.
+
+### Key Benefits
+
+**Ease of Deployment**: With this new functionality, you can quickly find and deploy the services you need, reducing the time and complexity of managing your cloud infrastructure. As a developer, if you are looking for a platform team sanctioned way to deploy a Kubernetes cluster, instead of chasing it down you can head to Pulumi Cloud, find the golden template and deploy it.
```
[the remark this answers is not in the document]
Comment (meagancojocar) [reply themes/default/content/blog/building-developer-portals/index.md:65]: I was worried I'm using too many lists haha, added lines between!

**Comment by scottslowe** [comment]
@meagancojocar Dropping a reminder not to publish until someone has gotten that GIF uploaded for you! :-)

File: themes/default/content/blog/building-developer-portals/index.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+- backstage
+
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+At Pulumi, we work with organizations that range from a few platform team members to entire departments for managing infrastructure. Many organizations, like [Mercedes-Benz](https://www.pulumi.com/case-studies/mercedes-benz/), have built internal developer platforms on top of Pulumi to enable developers to self-serve infrastructure templates, and partners like [AWS Proton](https://www.pulumi.com/blog/aws-proton-pulumi/) and [Port](https://www.getport.io/blog/using-pulumi-with-an-internal-developer-portal) have built integrations with Pulumi to enable self-service scenarios.
+
+Creating and distributing templates to ensure best practice cloud engineering for developers self-serving can be difficult to manage at scale. We are thrilled to have announced yesterday the launch of our new suite of tools to build internal developer portals with Pulumi, a result of working directly with our customers to understand their problems and how Pulumi can solve it.
```
Comment (interurban) [inline themes/default/content/blog/building-developer-portals/index.md:48]: For consideration as this first sentence tripped me up a bit. Perhaps something like:

Managing the creation and distribution of self-serve templates for cloud engineering best practices can be challenging at scale.

Comment (interurban) [reply themes/default/content/blog/building-developer-portals/index.md:48]: I'd also suggest the last part be "how Pulumi can solve them."

File: themes/default/content/blog/building-developer-portals/index.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+      description: My access token
+      secret: true
+```
+
+The above snippet includes an aws:region configuration value with a default value of us-west-2, as well as a myAccessToken secret without a default value.
+
+### Pulumi & Backstage
+
+We’ve seen many developer portal technologies rapidly growing in popularity over the last few years. In particular, we’ve seen Pulumi users adopting [Backstage](https://backstage.io) and as a result we built the Pulumi Backstage Plugin to address the needs of organizations using Backstage and Pulumi together. There are now two ways to use Pulumi from within Backstage:
+Pulumi Scaffolder Backend Module
```
Comment (interurban) [inline themes/default/content/blog/building-developer-portals/index.md:114]: Is this like option a, option b? in the preview link both these run into each other. should it be A and B or A or B?

File: themes/default/content/blog/building-developer-portals/index.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+    myAccessToken:
+      description: My access token
+      secret: true
+```
+
+The above snippet includes an aws:region configuration value with a default value of us-west-2, as well as a myAccessToken secret without a default value.
```
Comment (lukehoban) [inline themes/default/content/blog/building-developer-portals/index.md:109]:
```suggestion
The above snippet includes an `aws:region` configuration value with a default value of `us-west-2`, as well as a `myAccessToken` secret without a default value.
```

**Pull request merged**